packages/chat/src/ChatClient.ts

```diff
@@ -213,8 +216,9 @@ export class ChatClient extends IrcClient {
 	 *
 	 * @eventListener
 	 * @param channel The channel whose chat is cleared.
+	 * @param channelId The channel ID whose chat is cleared.
```

Comment:
"ID of the channel". It's not the ID that gets the chat cleared, it's the channel.